lib/rules/keyframe-selector-notation/__tests__/index.mjs

```diff
@@ -14,14 +14,18 @@ const ACCEPT_TIMELINE_RANGE_NAME_TEST_CASES = [
 		code: '@keyframes foo { entry 0% {} entry 100% {} }',
 		description: 'timeline-range-name entry',
 	},
-	{
-		code: '@keyframes foo { enter 0% {} enter 100% {} }',
-		description: 'timeline-range-name enter',
-	},
```

Comment:
I did not add a test with a warning for `enter 0% {}` because this rule does not aim to validate the named timeline ranges, only `from` vs. `0%`.

I think it is better to leave this as undefined behavior for now.

Reply:
🤔 Maybe better to omit it entirely from the changelog?

No one is writing `enter 10%` and this rule doesn't validate the named timeline ranges themselves.

I added it to the changelog before I fully understood why `enter` was allowed in this rule. Having a better understanding I think we can omit it.

-----

`enter` was at one point considered by the specification editors.
But as far as I know it was never implemented. So I think we can assume that no one is writing `@keyframes foo { enter 0% {} enter 100% {} }`.

Reply:
Understood. Thanks for the explanation. So, let's omit `enter` from the changelog. 👍🏼